pyproject.toml

```diff
@@ -27,7 +27,7 @@ copier = "copier.cli:CopierApp.run"
 "Bug Tracker" = "https://github.com/copier-org/copier/issues"
 
 [tool.poetry.dependencies]
-python = ">=3.6.2,<3.11"
+python = ">=3.6.2"
```

Comment:
According to https://endoflife.date/python, 3.6 has 6 days left of support, and I can safely assume we won't have copier v6 before that time, so you can put 3.7 as the lower limit.

Reply:
OK!